t/CR373909.csv

```diff
@@ -0,0 +1,146 @@
+purchase_time,currency_code,financial_market_bet_id,bet_type,buy_price,payout_price,sell_price,start_time,underlying_symbol,short_code,is_expired,sell_time
```

Comment:
Discussed with Jeff, we will create fake payout, buy_price data etc for test suite .